EIPS/eip-1962.md

```diff
@@ -0,0 +1,190 @@
+---
+eip: 1962
+title: EC arithmetics and pairings with runtime definitions
+author: Alex Vlasov (@shamatar)
+discussions-to: https://github.com/ethereum/EIPs/pull/1962
+type: Standards Track
+category: Core
+status: Draft
+created: 2019-04-22
+requires: 1109
+---
+
+# Simple summary
+
+This proposal is an extension and formalization of EIP1829 with an inclusion of pairings. EIP1109 is required due to possibly very low cost of point addition operation compared to the STATICCALL opcode.
```

Comment:
Can you make EIP1829 and EIP1109 links to https://eips.ethereum.org/EIPS/eip-??.md ?

Also can you quote opcodes, like STATICCALL, in the file?